addon/components/frost-bunsen-cell/component.js

```diff
@@ -1,12 +1,11 @@
 import _ from 'lodash'
-import Ember from 'ember'
 import computed, {readOnly} from 'ember-computed-decorators'
 
-import PropTypesMixin, {PropTypes} from 'ember-frost-bunsen/mixins/prop-types'
+import FrostComponent, {PropTypes} from 'ember-frost-component'
```

Comment:
I'm ok with FrostComponent because it's _our_ magic and as mentioned by @sandersky it's easy to figure out what the magic is by looking at the base component (and docs I assume) and the set of mixins included